### ISSUE TITLE
Test Site returns inconsistent footer links

### ISSUE THREAD

**Issue by agepoor** [facebook/Docusaurus#392]
### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md)?

Yes

### Environment

Windows, Node, npm

### Steps to Reproduce

1. > docusaurus-init
2.  Rename example folders to '/doc' and '/blog'
3. > cd website
4. Make no changes in siteConfig.js
5. > docusaurus-start

### Expected Behavior

The Test Site works, but I noticed the links in the footer are inconsistent.
For example:
When I click on "Getting Started (or other categories)" in the footer, on the Getting Started page the url of the exact same link changes from 'http://localhost:3000/test-site/docs/doc1.html' to 'http://localhost:3000/test-site/docs/en/doc1.html'. 

### Actual Behavior

The second link will be broken, because the docs are stored in '/docs' and not in '/docs/en'. I'm wondering what causes this 'inconsistent' footer behaviour.

**Comment by richardzcode** [comment]
Yes docs are stored in `docs` folder. But one major feature of Docusaurus is multi-language support. In runtime the doc is translated into multiple copies, one language each, depending on how many languages you enable in settings. 'en' is the default language.

My understanding is 'docs/doc1.html' and 'docs/en/doc1.html' should both exist. The link shouldn't be broken.

**Comment by agepoor** [comment]
To reproduce this:
1. `docusaurus-start`
2. Go to http://localhost:3000/
3. Click on footer link 'Getting Started (or other categories)' under Docs
4. You will be on http://localhost:3000/test-site/docs/doc1.html now
5. Click on 'Getting Started (or other categories)' under Docs again
6. It now returns 'Cannot GET /test-site/docs/en/doc1.html'

**Comment by em-cd** [comment]
I'm also having this issue, with a fresh Docusaurus site with no changes to the base configuration. Not sure if this is something to do with the Footer links, or the docs which aren't being moved into an `en` folder as I think the documentation says they should (https://docusaurus.io/docs/en/translation#translating-your-existing-docs)?
